routes/foodtrucks_test.go

```diff
@@ -95,3 +96,195 @@ func TestFoodTruckGetValid(t *testing.T) {
+	hours := [7][2]string{
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+	}
+	photos := []string{
+		"https://s3-media3.fl.yelpcdn.com/bphoto/d-1vKOqcEcRutpQ--jPa9A/o.jpg",
+		"https://s3-media4.fl.yelpcdn.com/bphoto/vXk0bXpV007fSWQiOTlHgg/o.jpg",
+		"https://s3-media2.fl.yelpcdn.com/bphoto/tUJ5gLnfRFhp_v-LUGj8Ww/o.jpg",
+	}
+	website := "www.google.com"
+	phone := "+18006729102"
+	description := "testDescription"
+	tags := []string{"food", "good food"}
+
+	// type addFoodTruckRequest struct {
+	// 	Name        *string       `json:"name"`
+	// 	Address     *string       `json:"address"`
+	// 	Location    [2]float64    `json:"location"`
+	// 	Hours       *[7][2]string `json:"hours"`
+	// 	Photos      *[]string     `json:"photos"`
+	// 	Website     string        `json:"website"`
+	// 	PhoneNumber string        `json:"phoneNumber"`
+	// 	Description string        `json:"description"`
+	// 	Tags        []string      `json:"tags"`
+	// }
+
+	newFoodTruckTest := addFoodTruckRequest{
+		Name:        &name,
+		Address:     &address,
+		Location:    location,
+		Hours:       &hours,
+		Photos:      &photos,
+		Website:     website,
+		PhoneNumber: phone,
+		Description: description,
+		Tags:        tags,
+	}
+
+	body, _ := json.Marshal(newFoodTruckTest)
+	req, _ := http.NewRequest("POST", "/foodtrucks", bytes.NewBuffer(body))
+	rr := httptest.NewRecorder()
+	handler := tests.AuthenticateMockUser(http.HandlerFunc(PostFoodTrucksHandler))
+	handler.ServeHTTP(rr, req)
+
+	expected := http.StatusOK
+	if rr.Code != expected {
+		t.Errorf("adding food truck expected status code of %v, but got %v", expected, rr.Code)
+	}
+
+	var addedFoodTruck models.JSONReview
+	json.NewDecoder(rr.Body).Decode(&addedFoodTruck)
+
+	updatedFoodTruck := tests.GetFoodTruck(addedFoodTruck.ID)
+	if updatedFoodTruck == nil || updatedFoodTruck.Name != "Luke's Coffee House" {
+		t.Error("Error finding the added food truck in the database.")
+		print(updatedFoodTruck.Name)
+
+		if updatedFoodTruck == nil {
+			t.Error("food truck is null")
+		}
+	}
+}
+
+func TestPostFoodTruckInvalidHours(t *testing.T) {
+	tests.ClearDB()
+
+	name := "Luke's Coffee House"
+	address := "2502 Nueces St\nAustin, TX 78705"
+	location := [2]float64{30.28793, -97.74731}
+	hours := [7][2]string{
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"10:00", "11:00"},
+		[2]string{"cute_string", "canthisbeparsed"},
+	}
+	photos := []string{
+		"https://s3-media3.fl.yelpcdn.com/bphoto/d-1vKOqcEcRutpQ--jPa9A/o.jpg",
+		"https://s3-media4.fl.yelpcdn.com/bphoto/vXk0bXpV007fSWQiOTlHgg/o.jpg",
+		"https://s3-media2.fl.yelpcdn.com/bphoto/tUJ5gLnfRFhp_v-LUGj8Ww/o.jpg",
+	}
+	website := "www.google.com"
+	phone := "+18006729102"
```

Comment:
i think this is technically the wrong format lol but it's ok, also it's not required